src/waitForImagesToRender.js

```diff
@@ -14,6 +14,16 @@ export default function waitForImagesToRender() {
       .filter(Boolean)
       .map(url => waitForImageToLoad(url));
 
+    document.body.querySelectorAll('*').forEach((element) => {
+      const computedStyle = window.getComputedStyle(element);
+      const bgImage = computedStyle.getPropertyValue('background-image');
+      if (bgImage && bgImage.startsWith('url(')) {
+        const url = computedStyle.getPropertyValue('background-image')
+          .replace(/^url\(['"]/, '').replace(/['"]?\)$/, '');
```

Comment:
This replace would be a bit better if it used a backreference.

Something like
```js
replace(/^url\((['"])(.+?)\1.*/, '$2')
```